propositions_and_proofs.md

```diff
@@ -13,10 +13,11 @@ One strategy for proving assertions about objects defined in the
 language of dependent type theory is to layer an assertion language
 and a proof language on top of the definition language. But there is
 no reason to multiply languages in this way: dependent type theory is
-flexible and expressive, and there is no reason we cannot represent
+flexible and expressive, and it turns out that there is no reason we cannot represent
 assertions and proofs in the same general framework.
 
-For example, we could introduce a new type, ``Prop``, to represent
+As an example of layering languages on top,
```

Comment:
Actually, now I'm not sure if this is meant as an example of layering languages or not; could spell it out more explicitly.